### PR TITLE
Add syscall clone3

### PR DESCRIPTION
**Pull request by StevenJiang1110**

### CODE REVIEW

File: regression/apps/scripts/process.sh [hunk 1]
```diff
@@ -8,7 +8,18 @@ SCRIPT_DIR=/regression
...
+# execve/execve
+# fork/fork
+# fork_c/fork
+# hello_pie/hello
+# hello_world/hello_world
+# pthread/pthread_test
+# pty/open_pty
+# signal_c/signal_test
+# "
+tests="pthread/pthread_test"
```
Comment (tatetian) [inline regression/apps/scripts/process.sh:22]: Why are the tests disabled?

**Review by tatetian — COMMENTED** [review]
Rebase this PR to the latest main

**Comment by StanPlatinum** [comment]
This PR will help to resolve #703 .

**Comment by tatetian** [comment]
@StevenJiang1110 Sorry, you have to rebase it again.

**Comment by tatetian** [comment]
@StevenJiang1110 Sorry, you have to rebase and fix the conflict again. Newly merged PRs cause conflict with this PR.

**Review by tatetian — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**